api.go

```diff
@@ -287,18 +287,25 @@ func apiSetDeploymentResults(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("OK"))
 }
 
+
+func addHeaders(req *http.Request) {
+	var token = services.Config().GetString("apigeesync_bearer_token")
+	req.Header.Add("Authorization", "Bearer "+token)
+}
+
+
 func transmitDeploymentResultsToServer(validResults apiDeploymentResults) error {
 
 	retryIn := bundleRetryDelay
 	maxBackOff := 5 * time.Minute
 	backOffFunc := createBackoff(retryIn, maxBackOff)
 
-	uri, err := url.Parse(apiServerBaseURI.String())
+	_, err := url.Parse(apiServerBaseURI.String())
 	if err != nil {
 		log.Errorf("unable to parse apiServerBaseURI %s: %v", apiServerBaseURI.String(), err)
 		return err
 	}
-	uri.Path = fmt.Sprintf("/clusters/%s/apids/%s/deployments", apidClusterID, apidInstanceID)
+	apiPath := fmt.Sprintf("%s/clusters/%s/apids/%s/deployments",apiServerBaseURI.String(), apidClusterID, apidInstanceID)
```

Comment:
Is this multi-return still? How does go handle multi-return functions in a string format like this?